packages/nextjs/src/utils/instrumentServer.ts

```diff
@@ -228,3 +285,16 @@ function shouldTraceRequest(url: string, publicDirFiles: Set<string>): boolean {
   // `static` is a deprecated but still-functional location for static resources
   return !url.startsWith('/_next/') && !url.startsWith('/static/') && !publicDirFiles.has(url);
 }
+
+function addRequestDataToEvent(event: SentryEvent, req: NextRequest): SentryEvent {
```

Comment:
This `add` method is actually setting the request data, not adding it; any data that was previously in the event will be overwritten. If the desired action is to set, I'd name the method `set`; and if the desired action is to add, update the implementation to add the data instead of setting it.

Reply:
There shouldn't be any request data prior to this, but you're right, why not be safe about it?